src/oscar/apps/dashboard/catalogue/views.py

```diff
@@ -636,7 +636,7 @@ class ProductLookupView(ObjectLookupView):
     model = Product
 
     def get_queryset(self):
-        return self.model.objects.browsable().all()
+        return self.model.objects.browsable_dashboard_recommended().all()
```

Comment:
`ProductLookupView` is a generic view that can be used by the `ProductSelect` and `ProductSelectMultiple` widgets in any context. It is not specific to recommended products and so this needs to remain generic. I think it's best just to use `model.objects.public()`.

Reply:
If its like this, ok. I broke something so I will start new PR with it.